### ISSUE TITLE
nomitmsitelist does not appear to work

### ISSUE THREAD

**Issue by opoplawski** [e2guardian/e2guardian#719]
In /etc/e2guardian/lists/example.group/nomitmsitelist:
```
apple.com
```
In /etc/e2guardian/e2guardian.conf:
```
.Define LISTDIR </etc/e2guardian/lists/example.group>
preauthstoryboard = '/etc/e2guardian/preauth.story'
sitelist = 'name=nomitm,path=__LISTDIR__/nomitmsitelist'
ipsitelist = 'name=nomitm,path=__LISTDIR__/nomitmsiteiplist'
```
preauth.story is the default version.

But I still get one of our certs when I connect via e2g with curl
```
* Server certificate:
*  subject: CN=apple.com
*  start date: Dec  6 13:35:51 2014 GMT
*  expire date: Dec  5 13:35:51 2024 GMT
*  subjectAltName: host "apple.com" matched cert's "apple.com"
*  issuer: C=US; ST=Washington; L=Redmond; O=Northwest Research Associates; emailAddress=admin@nwra.com; CN=proxy-ca
```
If I add it to exceptionsitelist then I get the upstream cert.  

Am I not understanding how nomitmsitelist is supposed to work?

**Comment by philipianpearce** [comment]
This may relate to #727.
Can you try latest branch release

**Comment by opoplawski** [comment]
That does look better, thanks!

**Comment by opoplawski** [comment]
While 5.4.4 references a fix for #727, it does not appear to be honoring nomitmsitelist for me.  Are we close to a 5.5 release?

**Comment by philipianpearce** [comment]
Yes,  near stable will be released today
